bindings/java/src/org/sleuthkit/datamodel/AbstractContent.java

```diff
@@ -391,6 +429,23 @@ public BlackboardArtifact getGenInfoArtifact() throws TskCoreException {
 		return getGenInfoArtifact(true);
 	}
 
+	/**
+	 * Returns the OS Account associated with this content if this is a file.
+	 * Otherwise, returns null.
+	 *
+	 * @return The OS Account ID associated with this content if this is a file.
+	 *         Otherwise, returns null.
+	 *
+	 * @throws TskCoreException
+	 */
+	private Long getOsAccountId() throws TskCoreException {
```

Comment:
I thought we weren't going to add this to Content?

Reply:
I'll take this out and resolve the merge conflict.